app/helpers/dashboards_helper.rb

```diff
@@ -18,4 +18,38 @@ def result_type(result)
   def status_tags(result)
     StatusTagService.run(resource: result)
   end
+
+  def result_date(result)
+    return if deactivated?(result)
+
+    return expired_date(result) if result.expired?
+    return will_expire_date(result) if result.is_a?(WasteCarriersEngine::Registration)
+
+    last_modified_date(result)
+  end
+
+  private
+
+  def deactivated?(result)
+    result.inactive? || result.refused? || result.revoked?
+  end
+
+  def expired_date(result)
+    I18n.t(".dashboards.index.results.date.expired", date: result.expires_on
+                                                                 .in_time_zone("London")
+                                                                 .strftime("%d/%m/%Y"))
```

Comment:
In WEX we've been using the convention of using Rails `time_formats.rb` to centralise our formats https://github.com/DEFRA/waste-exemptions-back-office/blob/master/config/initializers/time_formats.rb

Could we move this there and then switch to using `.to_formatted_s(:day_month_year)))`?

Reply:
Good shout. Tidied: https://github.com/DEFRA/waste-carriers-back-office/pull/387/commits/62ead7d9dddba5440b7ac8818f50a0adfa990b7e